autoconf/configure.in

```diff
@@ -4545,30 +4545,27 @@ AC_SUBST_FILE(DEBIAN_CONTROL_DIRECTOR_PYTHON_PLUGIN)
 dnl build a list of storage backends we need to build.
 BUILD_SD_BACKENDS=""
 if test x$use_libtool != xno; then
-   BUILD_SD_BACKENDS="${BUILD_SD_BACKENDS} libbareossd-fifo.la"
-   BUILD_SD_BACKENDS="${BUILD_SD_BACKENDS} libbareossd-gentape.la"
-   BUILD_SD_BACKENDS="${BUILD_SD_BACKENDS} libbareossd-tape.la"
-
-   if test X"$have_glusterfs" = "Xyes" ; then
-      BUILD_SD_BACKENDS="${BUILD_SD_BACKENDS} libbareossd-gfapi.la"
```

Comment:
libbareossd-gfapi.la got lost due to the cleanup here, causing build to fail. A fix is not required, as I already added it with an additional commit.

Reply:
I've been a bit optimistic on this. Compile still fails. I'll check for the reason later on.

Reply:
Strange thing compiled fine on my Linux install and it also compiled the stuff on the same platform which was used to build the customer rpms before. So could be some strange thing on OBS.

Reply:
Well, adding this AND regenerate "configure" (I missed that on the first try) solved the problem. So it does not seem to be OBS related.